### DISCUSSION TITLE
New logs actions (clear & mark)

### DISCUSSION THREAD

**Discussion by appaquet** [F1bonacc1/process-compose#208] [Ideas]
First, thanks for your tool! It has greatly improved my dev workflow.

Two actions that [k9s](https://k9scli.io/) offers when visualizing logs are the ability to clear logs and add a marker at the current position. Both are pretty useful for debugging services and removing clutter, and I’d love to see these features in process-compose.

Thanks!

**Comment by F1bonacc1** [comment]
Moved it to issues, to be added in one of the future releases.

**Reply by appaquet** [reply]
Thanks a lot !
